Sources/BraveWallet/Extensions/SolanaTxManagerProxyExtensions.swift

```diff
@@ -38,4 +38,25 @@ extension BraveWalletSolanaTxManagerProxy {
       }
     }
   }
+  
+  /// Fetches the estimatedTxFee for an array of transactions
+  @MainActor func estimatedTxFees(for transactions: [BraveWallet.TransactionInfo]
+  ) async -> [String: UInt64] {
```

Comment:
Is this needed? Can't we use the existing function above, or remove the function above and use this instead?